mos-platform/common/c/string.c

```diff
@@ -63,3 +63,70 @@ __attribute__((weak)) char *strrchr(const char *s, int c) {
       last = s;
   return (char *)last;
 }
+
+__attribute__((weak)) char* strrev(char* str)
+{
+    size_t len = strlen((const char*)str);
+    size_t i, j;
+    for (i = 0, j = len - 1; i < j; i++, j--)
+    {
+        char a = str[i];
+        str[i] = str[j];
+        str[j] = a;
+    }
+	return str;
```

Comment:
nit: decrease indent

Reply:
Forgot to run through clang-format. Done.